xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/Translations.xml

```diff
@@ -266,7 +266,7 @@ platform.wiki.sheet.prop.server=Alias
 platform.wiki.sheet.prop.homepage=Home page
 platform.wiki.sheet.prop.iswikitemplate=Wiki template
 platform.wiki.sheet.descriptor=This page defines how to access the \"{0}\" wiki, and some properties about it. For more configurations, please go to the
-platform.wiki.sheet.descriptor.admin=administraton page of the wiki.
+platform.wiki.sheet.descriptor.admin=Administration page of the wiki.
```

Comment:
It's usually a bad idea to change the case since you don't know how it's used. Usually it's made on purpose.

Reply:
Fixed.